lib/puppet-lint/optparser.rb

```diff
@@ -143,7 +143,7 @@ def self.build(args = [])
 
     unless args.include?('--no-config')
       opt_parser.load('/etc/puppet-lint.rc')
-      if ENV.key?('HOME') && File.readable?(ENV['HOME'])
+      if ENV.key?('HOME') && File.readable?(Dir.home)
```

Comment:
TIL: `Dir.home` exists. I think the `ENV.key?('HOME')` check no longer makes sense if you use `Dir.home`.